codalab/model/tables.py

```diff
@@ -431,7 +433,7 @@
     ),  # Whether the worker and the server have a shared filesystem.
     Column(
         'tag_exclusive', Boolean, nullable=False
-    ),  # Whether worker runs bundles if and only if they match tags.
+    ),  # Whether worker runs bundles that are untagged.
```

Comment:
Why did you change this? I thought `tag_exclusive` set to `True` means "the worker doesn't run untagged bundles" (whereas this description makes it sound like `True` = "the worker runs untagged bundles"). I think the earlier description is probably clearer.

Reply:
Good point, I'll leave it as is.